backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/chipgroup/multiple/internal/BpkMultiSelectChipGroupImpl.kt

```diff
@@ -85,13 +85,14 @@ internal fun BpkMultiSelectChipGroupImpl(
         BpkMultiChipGroupType.Wrap -> {
             FlowRow(
                 modifier = modifier.selectableGroup(),
+                horizontalArrangement = Arrangement.spacedBy(BpkSpacing.Md),
```

Comment:
now that we're on 1.5 we can set both the horizontal & vertical arrangement, so we can actually only apply the spacing internally as is required 🎉